libs/utils/utils.js

```diff
@@ -670,22 +682,48 @@ async function loadMartech({ persEnabled = false, persManifests = [] } = {}) {
     persEnabled,
     persManifests,
     utils: {
-      createTag, getConfig, getMetadata, loadLink, loadScript, updateConfig,
+      createTag,
+      getConfig,
+      getMetadata,
+      loadLink,
+      loadScript,
+      loadStyle,
+      updateConfig,
+      getPageSearchParams,
     },
   });
 
   return true;
 }
 
+function addPreviewToConfig() {
+  const searchParams = getPageSearchParams();
+  const mepOverride = searchParams.get('mep');
+  const mepMarker = searchParams.get('mepMarker');
+  const mepButton = searchParams.get('mepButton');
+  const previewPage = window.location.host.includes('.hlx.page') || window.location.host.includes('localhost');
+  const config = updateConfig({
+    ...getConfig(),
+    mep: {
+      preview: ((mepOverride !== null || mepMarker !== null || previewPage) && mepButton !== 'off'),
```

Comment:
Do we have to do a null check, or would presence be enough? e.g: 
`((mepOverride || mepMarker || previewPage) && mepButton !== 'off')`

Reply:
I had the exact some thought, but it resolves it to undefined.  Like it's interpreting it as a "if the first variable is undefined, use the next thing as a backup". But if the next value is also undefined then it resolves to undefined versus false.  Maybe I'm missing something.

Reply:
You could convert that `undefined` to bool via:
`preview: !!((mepOverride || mepMarker || previewPage) && mepButton !== 'off'),`